guard/templates/guard/guardDeployment.yaml

```diff
@@ -250,5 +250,9 @@ spec:
             value: "5000"
           - name: PERSON_WITH_ESCORT_PAIR_CACHE_LIFESPAN
             value: "5000"
+
           - name: FCM_ALL_TOPIC
-            value: "allProd"
\ No newline at end of file
+            valueFrom:
+              secretKeyRef:
+                name: firebase-secret-terraform
+                key: TOPIC_NAME
```

Comment:
could be more specific name, like FIREBASE_NOTIF_TOPIC_NAME but i guess ok do not change it